omod-2.0/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs2_0/ConceptAttributeTypeController2_0Test.java

```diff
@@ -73,4 +74,20 @@ public void shouldCreateConceptAttributeType() throws Exception {
 
     }
 
+    @Test
+    public void shouldUpdateConceptAttributeType() throws Exception {
+        String existingConceptAttributeTypeUuid ="9516cc50-6f9f-11e0-8414-001e378eb67e";
```

Comment:
Can you also take a look at our naming conventions?

Reply:
have looked through .... by using `conceptAttributeTypeUuid ="9516cc50-6f9f-11e0-8414-001e378eb67e";` would it be better or Otherwise @dkayiwa  any Suggestion

Reply:
Here they are: https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-NamingConventions

Reply:
Got it .....but still to me not clear though

Reply:
If not clear, then what do you mean by `Got it`?